CHANGELOG.md

```diff
@@ -5,10 +5,15 @@
 ### Breaking changes
 
 - [\#56](https://github.com/arkworks-rs/crypto-primitives/pull/56) Compress the output of the Bowe-Hopwood-Pedersen CRH to a single field element, in line with the Zcash specification.
+- [\#60](https://github.com/arkworks-rs/crypto-primitives/pull/60) For byte based CRHs, `CRHGadget::evaluate` takes `&Vec<UInt8>` instead of `[UInt8]`.
```

Comment:
Why is this? The two are equivalent, no? Unless you mean that it takes `Vec<UInt8>`

Reply:
I've removed this line in today's commit because I updated the relevant APIs